src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/DefaultTraceListener.cs

```diff
@@ -51,7 +51,7 @@ public string LogFileName
             get
             {
                 if (!_settingsInitialized) InitializeSettings();
-                return _logFileName;
+                return _logFileName!;
```

Comment:
What is ensuring this is non-null? A public consumer can set it to `null`.

Reply:
`InitializeSettings();` inits it and sets `_settingsInitialized = true`